src/main/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManager.java

```diff
@@ -347,8 +360,14 @@ private void updateBindingIfNecessary(@Nullable WorkspaceFolderWrapper folder, M
     }
   }
 
-  private static void updateGlobalStorageAndLogResults(ServerConfiguration serverConfiguration, ConnectedSonarLintEngine engine) {
-    UpdateResult updateResult = engine.update(serverConfiguration, null);
-    LOG.info("Global storage status: {}", updateResult.status());
+  private static void updateGlobalStorageAndLogResults(ServerConfiguration serverConfiguration, ConnectedSonarLintEngine engine, List<String> servers, String server) {
```

Comment:
I would rename `servers` to `failedServerIds` and `server` to `serverId`

Reply:
Done.